src/main/java/com/carrotcreative/cream/util/HashingUtil.java

```diff
@@ -0,0 +1,52 @@
+package com.carrotcreative.cream.util;
+
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import android.support.annotation.NonNull;
+
+public class HashingUtil
+{
+    private static final int CACHE_SIZE = 50;
```

Comment:
50 of what?

I see an example on http://developer.android.com/reference/android/util/LruCache.html, and it looks like this:

`int cacheSize = 4 * 1024 * 1024; // 4MiB`